test/integration/features/generic-driver.feature

```diff
@@ -0,0 +1,49 @@
+@generic-driver
+Feature: With generic driver Minishift can provision remote unprovisioned VM 
+
+  Scenario: User creates an unprovisioned VM
+    Given setting up environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" with value "y" succeeds
+    When executing "minishift profile set remote" succeeds
+    And executing "minishift start --no-provision" succeeds
+    Then Minishift should have state "Running"
+
+  Scenario: Provisioning remote instance
+    Given unset environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" succeeds
+    And setting scenario variable "REMOTE_VM_IP" to the stdout from executing "minishift ip"
+    And executing "minishift profile set minishift" succeeds
+    When executing "minishift start --vm-driver generic --remote-ipaddress $(REMOTE_VM_IP) --remote-ssh-key .minishift/profiles/remote/machines/remote/id_rsa --remote-ssh-user docker" succeeds
+    Then Minishift should have state "Running"
+
+  Scenario: Stopping with generic driver
+  Stopping with generic driver puts oc cluster down, remote VM however keeps running.
+    When executing "minishift stop" succeeds
+    And executing "minishift status" succeeds
+    Then stdout should match "OpenShift:\s+Stopped"
+
+  Scenario: VM at remote location is still running
+    When executing "minishift status --profile remote" succeeds
+    Then stdout should match "Minishift:\s+Running"
+    And stdout should match "OpenShift:\s+Stopped"
+
+  Scenario: Starting stopped cluster with generic driver
+    When executing "minishift start" succeeds
+    Then executing "minishift status" succeeds
+    And stdout should match "Minishift:\s+Running"
+    And stdout should match "OpenShift:\s+Running"
+  
+  Scenario: Remote VM is running and cluster is up
+    When executing "minishift status --profile remote" succeeds
+    Then stdout should match "Minishift:\s+Running"
+    And stdout should match "OpenShift:\s+Running"
```

Comment:
Wondering how this is in running state because nowhere in previous step it is being started

Reply:
@amitkrout It is started at line 7.

Reply:
You started it on remote profile and then at line no 19 it is stopped

Reply:
But at line 29 I start the OpenShift again with generic driver, so here both VM (generic does not affect that at all) and the OpenShift are running on the remote machine.

Reply:
> But at line 29 I start the OpenShift again with generic driver

How? Ideally, we are not saving any flags like ```--vm-driver```, ```--remote-ipaddress``` ...in the config or even in env. Then how ```minishift start``` will consider these variable.